db/structure.sql

```diff
@@ -685,7 +692,7 @@ CREATE TABLE public.oban_jobs (
     attempted_by text[],
     discarded_at timestamp without time zone,
     priority integer DEFAULT 0 NOT NULL,
-    tags text[] DEFAULT ARRAY[]::text[],
+    tags character varying(255)[] DEFAULT ARRAY[]::character varying[],
```

Comment:
the schema from structure.sql is out of sync across dev envs and it causes this record type to be different. I recently recreated the dev and test DBs locally starting from the Phoenix app and I have the `text[]` field type on tags for this oban table that has nothing to do with the PR. Stone has the same issue. the TLDR is that we don't want to commit these changes, so either don't `git add` this file, or if you have changes to commit you need to commit those lines individually using `git add -p`. you can resolve this problem so it's not showing up in the diff by dropping the DB and following the steps on the Phoenix app to recreate it, then running the new migrations on rails. after that you shouldn't see this line change. I'm not sure about the other comment above, but it's probably fine either way. I just don't like to have to remember this everytime. I will revert this change from this PR.

Reply:
@stepchud Okay thanks for letting me know, I'll keep this in mind and recreate my db 👍

Reply:
Ok let me know if it comes back when you recreate the DB. It might be a difference in the Oban version we have installed locally.